modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/reassign/single/SingleReassignModal.es.js

```diff
@@ -64,27 +65,22 @@ const SingleReassignModal = () => {
 	});
 
 	const reassignButtonHandler = useCallback(() => {
-		if (assigneeId && taskId) {
-			setErrorToast(() => false);
-			setSendingPost(() => true);
-			postData()
-				.then(() => {
-					onClose();
-					toaster.success(
-						Liferay.Language.get('this-task-has-been-reassigned')
-					);
-					setErrorToast(false);
-					setSendingPost(false);
-					setSelectedItem({});
-				})
-				.catch(() => {
-					setErrorToast(true);
-					setSendingPost(false);
-				});
-		}
-		else {
-			onClose();
-		}
+		setErrorToast(() => false);
+		setSendingPost(() => true);
```

Comment:
What do you think about to use only setState(newValue) instead of setState(() => newValue)?